cluster/mesos/task.go

```diff
@@ -184,7 +184,11 @@ func (t *task) monitor() (bool, []byte, error) {
 	case mesosproto.TaskState_TASK_FINISHED:
 		return true, taskStatus.Data, nil
 	case mesosproto.TaskState_TASK_FAILED:
-		return true, nil, errors.New(taskStatus.GetMessage())
+		errorMessage := taskStatus.GetMessage()
+		if errorMessage == "Abnormal executor termination" {
+			errorMessage += " : please verify your SWARM_MESOS_USER is correctly set"
```

Comment:
I had the same thought but this is a quick improvement for a most known occurring error. Also this practice can be found in other places in Docker: https://github.com/docker/docker/blob/master/api/client/utils.go#L104